.github/workflows/ci.yml

```diff
@@ -1,100 +1,20 @@
-    steps:
-      - uses: actions/checkout@v2
-      - name: Setup ruby
-        uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: '2.7'
-          bundler-cache: true
-      - name: Run rake check
-        run: bundle exec rake check
-      - name: Run rake validate
-        run: bundle exec rake validate
-      - name: Run rake lint
-        run: bundle exec rake lint
-      - name: Setup Test Matrix
-        id: get_outputs
-        run: bundle exec metadata2gha --use-fqdn --pidfile-workaround false
-
-  unit:
-    needs: setup_matrix
-    runs-on: ubuntu-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        include: ${{fromJson(needs.setup_matrix.outputs.puppet_unit_test_matrix)}}
-    env:
-      BUNDLE_WITHOUT: development:system_tests:release
-      PUPPET_GEM_VERSION: "~> ${{ matrix.puppet }}.24"
-    name: Unit / Puppet ${{ matrix.puppet }} (Ruby ${{ matrix.ruby }})
-    steps:
-      - uses: actions/checkout@v2
-      - name: Setup ruby
-        uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: ${{ matrix.ruby }}
-          bundler-cache: true
-          cache-version: '1'
-      - name: Run tests
-        run: bundle exec rake parallel_spec
 
-  acceptance:
-    needs: setup_matrix
-    runs-on: ubuntu-latest
-    env:
-      BUNDLE_WITHOUT: development:test:release
-      PUPPET_GEM_VERSION: '~> 7.24'
-    strategy:
-      fail-fast: false
-      matrix:
-        setfile: ${{fromJson(needs.setup_matrix.outputs.beaker_setfiles)}}
-        puppet: ${{fromJson(needs.setup_matrix.outputs.puppet_major_versions)}}
-        pulpcore_version:
-          - '3.28'
-          - '3.22'
-          - '3.21'
-    name: Acceptance / ${{ matrix.puppet.name }} - ${{ matrix.setfile.name }} - Pulp ${{ matrix.pulpcore_version }}
-    steps:
-      - uses: actions/checkout@v2
-      - name: Setup ruby
-        uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: '2.7'
-          bundler-cache: true
-          cache-version: '1'
-      - name: Run tests
-        run: bundle exec rake beaker
-        env:
-          BEAKER_PUPPET_COLLECTION: ${{ matrix.puppet.collection }}
-          BEAKER_setfile: ${{ matrix.setfile.value }}
-          # In Puppet 7 the locale ends up being C.UTF-8 if it isn't passed.
-          # This locale doesn't exist in EL7 and won't be supported either.
-          LANG: en_US.UTF-8
-          BEAKER_FACTER_PULPCORE_VERSION: ${{ matrix.pulpcore_version }}
+concurrency:
+  group: ${{ github.ref_name }}
+  cancel-in-progress: true
 
-  tests:
-    needs:
-      - unit
-      - acceptance
-    runs-on: ubuntu-latest
-    name: Test suite
-    steps:
-      - run: echo Test suite completed
+jobs:
+  puppet:
+    name: Puppet
+    uses: ekohl/gha-puppet/.github/workflows/beaker.yml@set-in-env-var
+    with:
+      pidfile_workaround: 'false'
+      rubocop: false
+      cache-version: '1'
+      beaker_facter: '--beaker-facter pulpcore_version:Pulp:3.28,3.22,3.21'
```

Comment:
this feels odd api wise.
either: `m2gha_options: --beaker-facter pulpcore_version:Pulp:3.28,3.22,3.21`
or: `beaker_facter: ' pulpcore_version:Pulp:3.28,3.22,3.21'`
wdyt?